frontend/src/components/custom-tools/prompt-card/Header.jsx

```diff
@@ -157,30 +158,34 @@ function Header({
                 <PlayCircleOutlined className="prompt-card-actions-head" />
               </Button>
             </Tooltip>
-            <Tooltip title="Run All">
-              <Button
-                size="small"
-                type="text"
-                className="prompt-card-action-button"
-                onClick={() => handleRunBtnClick()}
-                disabled={
-                  (updateStatus?.promptId === promptDetails?.prompt_id &&
-                    updateStatus?.status ===
-                      promptStudioUpdateStatus?.isUpdating) ||
-                  disableLlmOrDocChange?.includes(promptDetails?.prompt_id) ||
-                  indexDocs?.includes(selectedDoc?.document_id)
-                }
-              >
-                <PlayCircleFilled className="prompt-card-actions-head" />
-              </Button>
-            </Tooltip>
+            {!isSimplePromptStudio && (
+              <Tooltip title="Run All">
+                <Button
+                  size="small"
+                  type="text"
+                  className="prompt-card-action-button"
+                  onClick={() => handleRunBtnClick()}
```

Comment:
I think this change exist from my PR. can please change remove that arrow func here

Reply:
Done, @jagadeeswaran-zipstack